build/SharedFx.targets

```diff
@@ -133,4 +270,6 @@
       WorkingDirectory="$(SharedFxOutputDirectory)"
       Overwrite="true"/>
   </Target>
+
+  <Target Name="GenerateSharedFx" DependsOnTargets="BuildSharedFx;CrossGenAssemblies;CrossGenSymbols;PackSharedFx"/>
```

Comment:
Requires a CI config change, right? Could you keep BuildSharedFx as the top-level target and rename the existing BuildSharedFx to something like "CollectSharedFxFiles"

Reply:
Sure